lib/commands/order/reminder.rb

```diff
@@ -23,10 +23,20 @@ def run
       return "You are not the foreman!" if !Apprentice.foreman?(@user_id)
 
       @mark_all_out.update
-      not_ordered_members = CustomerProvider.new.customers_without_order
 
-      return "Everyone has an order." if not_ordered_members.empty?
-      not_ordered_members.join("\n").strip
+      crafter_to_remind = Order.crafter_without_order(@requester.office).map do |crafter|
```

Comment:
if `crafter_to_remind` is going to contain a collection, could we name it `crafters_to_remind`?

Reply:
Good one